health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/MyMicroplans.js

```
@@ -0,0 +1,52 @@
+const MyMicroplans = () => {
+  const { t } = useTranslation();
+  const [defaultValues, setDefaultValues] = useState(defaultSearchValues); // State to hold default values for search fields
+  const history = useHistory(); 
+  useEffect(() => {
+    // Set default values when component mounts
+    setDefaultValues(defaultSearchValues);
+  }, []);
+
+  const onTabChange = (n) => {
+    
+    setTabData((prev) => prev.map((i, c) => ({ ...i, active: c === n ? true : false }))); //setting tab enable which is being clicked
```

Biome

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)